.github/workflows/db-test.yml

```diff
@@ -0,0 +1,32 @@
+name: PostgreSQL Test
+
+on:
+  release:
+    types: [published]
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - 'db_scripts/**'
```

Comment:
also add test section for upgrade scripts...
this one can be merged for time being.